Extension/package.json

```diff
@@ -461,6 +461,12 @@
           "markdownDescription": "%c_cpp.configuration.codeAnalysis.clangTidy.args.markdownDescription%",
           "scope": "resource"
         },
+        "C_Cpp.codeAnalysis.clangTidy.useDashP": {
```

Comment:
I'm having a hard time with the name.  Can it be `useBuildPath` (the full name of the argument) instead?  The description doesn't need to change.

Reply:
Or maybe `useCompileCommandsPath`?

Reply:
Sure, that is fine, which should I use?

Reply:
Whichever you like better.  I don't think I have a strong preference.

Reply:
On potential downside of using "useBuildPath" is that there's a possible confusion if users don't read the description because it still reads/uses the build path if compileCommands is set, but it's read by IntelliSense code. But I'm not sure if that will actually lead to any real confusion.